src/Test.UnitTests.Sarif.Converters/TestData/NessusConverter/ExpectedOutputs/ValidResults.sarif

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
@ejohn20 can you please also share a sample Nessus Sarif file and a sample CIS CAT Sarif file with real world rules/results data, so team can review them on different viewers and provide feedbacks?

Reply:
@yongyan-gh - Yes I'll provide them to you via email. Thanks.